### PR TITLE
[PD-3261] export current tab

### PR DESCRIPTION
**Pull request by mwarnsleyDE**
## What's this PR do?
PR fixes the exporting for the current tab that the user is on inside of analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of any of the analytics reports

## How should this be manually tested?
Pull down repository. Go into any one of the analytics and create more tabs by drilling down. Switch the tabs and export the data which should be the data for the current tab

### CODE REVIEW

File: gulp/src/analytics/actions/header-actions.js [hunk 1]
```diff
@@ -4,12 +4,18 @@ export function doExportTab(presentationType) {
...
       const link = document.createElement('a');
+      const tabs = getState().pageLoadData.navigation;
       const currentReport = getState().pageLoadData.activeReport;
       const storedDates = {
         startDate: getState().pageLoadData.globalStartDate,
         endDate: getState().pageLoadData.globalEndDate,
       };
-      const storedFilters = getState().pageLoadData.activeFilters;
+      let storedFilters = [];
+      for (let f = 0; f < tabs.length; f++) {
```
Comment (aljordan82) [inline gulp/src/analytics/actions/header-actions.js]: It is not clear what this loop does.

Comment (mwarnsleyDE) [inline gulp/src/analytics/actions/header-actions.js]: @aljordan82 this loops through the tabs themselves and finds the active one that you are currently on and sets it. Before we were just doing the last tab that was created but now it does the currently active tab that you are on

Comment (aljordan82) [inline gulp/src/analytics/actions/header-actions.js]: Ideally, I would use 'find'
`var storedFilters = tabs.find((tab) => tab.active).activeFilters || []`

**Comment by tdphillips** [comment]
Have the requested changes been made? I suspect so but I'd rather not merge without being certain.

@aljordan82, @mwarnsleyDE

**Comment by mwarnsleyDE** [comment]
@tdphillips @aljordan82 I've made the changes requested

**Pull request merged**